x-pack/test/saved_object_api_integration/common/config.ts

```diff
@@ -56,7 +56,7 @@ export function createTestConfig(name: string, options: CreateTestConfigOptions)
         serverArgs: [
           ...config.xpack.api.get('kbnTestServer.serverArgs'),
           '--server.xsrf.disableProtection=true',
-          `--plugin-path=${path.join(__dirname, 'fixtures', 'saved_object_test_plugin')}`,
+          `--plugin-path=${path.join(__dirname, 'plugins/saved_object_test_plugin')}`,
```

Comment:
It feels odd and potentially error prone to use `path.join` for part of the directory path but not the whole one. Why not do:

```suggestion
          `--plugin-path=${path.join(__dirname, 'plugins', 'saved_object_test_plugin')}`,
```

Reply:
Mostly because many of these uses wrapped weirdly and having directories "exploded" like that makes it really hard for me to read the path. We don't support windows development and I would normally use `path.resolve` which takes care of converting path separators in case we are running on windows. Would you prefer that I make a specific change here?

Reply:
Went with `path.resolve(), hope you don't mind.